src/pl/plpython/plpy_spi.c

```diff
@@ -54,11 +55,15 @@ PLy_spi_prepare(PyObject *self, PyObject *args)
 	{
 		PLy_exception_set(PyExc_TypeError,
 					   "second argument of plpy.prepare must be a sequence");
+		PLy_enter_python_intepreter = true;
```

Comment:
lack PLy_enter_python_intepreter= false at the beginning of spi prepare

Reply:
Thanks, fixed